thirdparty/README.md

```diff
@@ -480,6 +480,20 @@ Files extracted from upstream source:
 - License.txt
 
 
+## Rvo2-3D
+
+- Upstream: http://gamma.cs.unc.edu/RVO2/
+- version: 1.0.1
+- License: Apache 2.0
+
+Files extracted from upstream source:
+
+- All .cpp and .h files in the `src/` folder except for RVOSimulator.cpp and RVOSimulator.h
+- LICENSE
+- README
```

Comment:
We don't need their README (also removed in `master`).

Reply:
Do you want me to remove the readme in the directory as well? Because it does still exist in master.

Reply:
Yeah you can remove it here.  We can remove it in master too after this has been merged, I was planning to suggest you look at updating rvo2 and recast after this ;)